client/src/components/CompletedProfile/CompletedProfile.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import styled from "styled-components";
+import { theme, mq } from "constants/theme";
+import BaseButton from "components/Button/BaseButton";
+
+const { tablet } = mq;
+
+// ICONS
+const { colors, typography } = theme;
+
+export const ProfileCompletedWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  background-color: #eeeeef;
```

Comment:
I think this one must be constants.colors.OFF_WHITE

Reply:
yep try to avoid hard coded values

Reply:
Right! Got it 😀